### ISSUE TITLE
Feature request: hotfix log filtering

### ISSUE THREAD

**Issue by Shauren** [Marlamin/wow.tools#77]
Would be nice to be able to filter hotfix log page by wow branch (retail/classic/ptr)

Alternative filtering mode: version excluding build number (for example: 9.0.5, 9.0.2) - if old data is available

Filtering by exact build number is less useful, most (or all?) 9.0.5 builds should have the same hotfix list (not put into game files directly)

**Comment by Marlamin** [comment]
Yeah, I do want to improve the hotfix page (and pipeline) but right now it is very build number focused. Planned for the future, will keep this open.
